scripts/azcli-integration/main.tf

```diff
@@ -48,7 +48,7 @@ output "test_gen1_name" {
 }
 
 output "test_storage_account_key" {
-  value = module.this.test_storage_account_key
+  value     = module.this.test_storage_account_key
```

Comment:
formatting changes are irrelevant for contents of this PR. Please revert and make a separate PR for formatting changes if you think they are needed.